README.rst

```diff
@@ -27,8 +27,16 @@ provider) where the strings can be translated.
 Compile translated strings by running `make compile_translations`. This will produce a .mo file for each .po in the repo.
 The .mo files are read by Django and are used to provide translations in the running application.
 
-Testing credentials-themes changes in devstack
-----------------------------------------------
-See instructions_ on the OpenEdX Wiki.
-
-.. _instructions: https://openedx.atlassian.net/wiki/spaces/SOL/pages/608698737/Testing+WL+themes+in+devstack
+Developing in devstack
+----------------------
+In order to develop in devstack, make sure your devstack is running, then:
+
+#. Switch your devstack to the `edX theme`_
+#. Clone this repo into ``<devstack_folder>/src/``
+#. ``cd <devstack_folder>/devstack/``
+#. ``make credentials-shell``
+#. ``pip install -e /edx/src/credentials-themes``
+#. ``make requirements``
+#. ``make static``
+
+.. _`edX theme`: https://openedx.atlassian.net/wiki/spaces/microb/pages/2047673326/Switching+the+theme+in+edx-platform
```

Comment:
Nit: I believe `credentials-themes` is an open-source repo, but I don't believe our space is available by the public.

I don't actually believe that we are going to have a lot of OSPRs coming out of this repo... but I thought it might be weird having a potentially broken link here?

Reply:
Yeah, I wondered about that. Higher up in this file it says "_This information is NOT LICENSED for usage by others._" so I thought this was fine, at least for now.

I could be convinced to move the wiki page...

Reply:
Ah, yes, I see that disclaimer now (and it makes me feel better). I don't think it is worth the effort to move the page.